loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -56,6 +56,29 @@ interface IRegisteredExtension {
     factory: (loader: GLTFLoader) => IGLTFLoaderExtension;
 }
 
+/**
+ * Type of data held by a texture
+ * @hidden
+ */
+export enum TextureDataType {
+    /** color data (albedo, emissive, ...) */
+    Color,
+    /** roughness data */
+    Roughness,
+    /** normal map */
+    Normal,
+    /** glossiness data */
+    Glossiness,
+    /** specular map */
+    Specular,
+    /** transmission map (thickness) */
+    Transmission,
+    /** metallic/roughness data */
+    MetallicRoughness,
+    /** occlusion data */
+    Occlusion
+}
```

Comment:
I'm not sure why this enum is necessary. The code only needs to know whether the texture is color data or not. Can we just use a boolean?

Reply:
I had done this instead of the bool in case we would need a more precise information in the future.

I have reverted to use a bool.

Reply:
We don't know what kind of textures there will be in the future. This enum includes entries from extensions which works for uilt-in extensions but can't be updated in custom extensions. Anyways, you've already updated to a bool, so we should be good.